core/src/main/java/org/apache/accumulo/core/client/ScannerBase.java

```diff
@@ -44,6 +44,17 @@
  */
 public interface ScannerBase extends Iterable<Entry<Key,Value>>, AutoCloseable {
 
+  /**
+   * Consistency level for the scanner. The default level is IMMEDIATE, which means that this
```

Comment:
It's not just data that has not been minor compacted. The ScanServer contains a cache of metadata entries. If the metadata entries are cached for a Tablet, then when the ScanServer creates a SnapshotTablet, it will be from those files, not necessarily the up-to-date list of files in the metadata table. The tablet metadata cache expiration is set using Property.SSERV_CACHED_TABLET_METADATA_EXPIRATION.